hs/CalendarResponse.hs

```diff
@@ -0,0 +1,281 @@
+module CalendarResponse where
+
+import Data.List (sort)
+import Data.List.Split (splitOn)
+import Data.Time (Day, UTCTime, addDays, formatTime, getCurrentTime)
+import Happstack.Server (ServerPart, Response, toResponse)
+import Control.Monad.IO.Class (liftIO)
+import System.Locale
+import Database.CourseQueries (returnTutorialTimes, returnLectureTimes)
+import qualified Data.Text as T
+import Database.Tables as Tables
```

Comment:
I will use 'import Database.Tables (Time, timeField)' instead. I imported Database.Tables as Tables because I thought that was the standard way for importing files that had to with the database. However, now that I think about it, such importing just adds readability in files such as courseQueries. Please let me know if you agree with the new way I am planning to use to import Database.Tables @Ian-Stewart-Binks.